docs/fsharp/language-reference/arrays.md

```diff
@@ -3,7 +3,7 @@ title: Arrays
 description: Learn how to create and use arrays in the F# programming language.
 ms.date: 05/16/2016
 ---
-# Arrays
+# Arrays in F\#
```

Comment:
Do you want me to add this suffix to all articles in F# reference for consistency?

Reply:
I don't think so, but pinging @cartermp for his opinion.

Reply:
Sure!